src/Symfony/Component/ObjectMapper/Tests/ObjectMapperTest.php

```diff
@@ -211,16 +211,16 @@ public function get(string $id): mixed
         };
     }
 
-    public function testSourceOnly(): void
+    public function testSourceOnly()
     {
-        $a = new \stdClass();
+        $a = new \stdClass;
```

Comment:
Are we sure about this one?

Reply:
Strange to me as well

Reply:
This is inconsistent in this file. Everywhere in the Symfony codebase we use the parenthesis. It can be updated the other way.
```suggestion
        $a = new \stdClass();
```